package.json

```diff
@@ -116,7 +116,7 @@
     "sift": "^3.2.7",
     "style-loader": "^0.16.1",
     "superagent": "^3.5.2",
-    "svgo": "^0.7.2",
+    "svg-url-loader": "^2.0.2",
```

Comment:
We are not distributing that source code - same way I don't see Webpack here https://help.apiary.io/faq/licensing_information/

I'll try to get a resolution on TL Sync today, don't feel it's blocking tho